.github/workflows/integrations.yml

```diff
@@ -208,12 +208,19 @@ jobs:
           - os: windows-2019
             preamble: call "C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Enterprise\\VC\\Auxiliary\\Build\\vcvarsall.bat" x64
             site-packages: Lib/site-packages
+            python_version: "3.9"
             shell: cmd
           - os: ubuntu-20.04
             site-packages: lib/python3.9/site-packages
+            python_version: "3.9"
+            shell: bash
+          - os: ubuntu-20.04
+            site-packages: lib/python3.10/site-packages
+            python_version: "3.10"  # CY23
```

Comment:
Q: do you support CY24, Python 3.11?

Asking as it's relevant to Nuke 16.

Reply:
Yeah we build wheels for 3.9, 3.10 and 3.11 and publish them to PyPI (`build-wheels.yml`, `deploy-pypi.yml`)

Reply:
I asked the question as in this snippet, I only see Python 3.9 and 3.10 mentioned for Linux. Should 3.11 be here too?

Reply:
Oh that was just to keep the build matrix down and reduce CI wait time, by focusing testing just over each side of the ABI fence.

Ideally yeah every CI test would run on every platform for every Python version, but pragmatically it adds a lot of wait time to CI for not much extra coverage. Other Python versions are tested in other CI jobs, albeit less thoroughly (just unit tests).

Reply:
Understood!
> by focusing testing just over each side of the ABI fence.